terraform/rds.tf

```diff
@@ -0,0 +1,43 @@
+module "db" {
+  source = "terraform-aws-modules/rds/aws"
+  version = "6.8.0"
+
+  identifier = "database-${var.environment}-${var.project_name}"
+
+  engine            = "mysql"
+  engine_version    = "8.0.35"
+  instance_class    = "db.t3.micro"
+  allocated_storage = 5
+
+  db_name  = "products"
+  username = "ujwal"
+  password = "ujwal123"
```

Comment:
use relevant aws resource to store these secrets.

Reply:
used secrets manager